### PR TITLE
Removed PathsDocIT test depending on order of paths

### PR DESCRIPTION
**Pull request by burqen**
The order in which paths are returned from Dijkstra is not important. Should not test on that.
Also added ascii art to tests.

### CODE REVIEW

File: community/server/src/test/java/org/neo4j/server/rest/PathsDocIT.java [hunk 1]
```diff
@@ -202,8 +242,6 @@ public void shouldGetCorrectDijkstraPathsWithWeights() throws Exception
         assertEquals( 1.5, secondPath.get( "weight" ) );
         // 5 = 3 +2
         assertEquals( 5, (Integer) firstPath.get( "length" ) + (Integer) secondPath.get( "length" ) );
-        assertThatPathHasLength( firstPath, 3 );
-        assertThatPathHasLength( secondPath, 2 );
```
Comment (davidegrohmann) [inline community/server/src/test/java/org/neo4j/server/rest/PathsDocIT.java]: ?

Comment (davidegrohmann) [inline community/server/src/test/java/org/neo4j/server/rest/PathsDocIT.java]: What about adding: `assertEquals( 1, Math.abs( (Integer) firstPath.get( "length" ) - (Integer) secondPath.get( "length" ) ) );` and a comment explaining that the assertions try not to rely on the order of results?

Comment (burqen) [inline community/server/src/test/java/org/neo4j/server/rest/PathsDocIT.java]: We could do that, but the information we would get is already checked in `assertEquals( 5, (Integer) firstPath.get( "length" ) + (Integer) secondPath.get( "length" ) )` right?

Comment (davidegrohmann) [inline community/server/src/test/java/org/neo4j/server/rest/PathsDocIT.java]: You might get back paths of length `1` and `4` and your assertion won't fail.

Comment (burqen) [inline community/server/src/test/java/org/neo4j/server/rest/PathsDocIT.java]: If that is the case then something is seriously broken, but yeah, better safe than sorry I suppose.
Fixed.

**Comment by burqen** [comment]
@davidegrohmann Wanna have another look?

**Pull request merged**